packages/data/addon/models/metadata/metric.ts

```diff
@@ -42,25 +45,25 @@ export default class Metric extends Column {
   }
 
   /**
-   * @property {Boolean} hasParameters
+   * @property {boolean} hasParameters
    */
-  get hasParameters() {
+  get hasParameters(): boolean {
     return !!this.arguments?.length;
   }
 
   /**
-   * @property {Object[]} arguments - arguments for the metric
+   * @property {FunctionArgument[]} arguments - arguments for the metric
```

Comment:
Navi doesn't currently create instances of `FunctionArgument` because no API has implemented support for it yet. That's why the type was documented as a plain object. Does forcing this property to a list of FunctionArguments cause any problems?

Reply:
I'll revert it for now until we implement it